reference_docs/YAML_models/standoc_document_attributes.yaml

```diff
@@ -0,0 +1,724 @@
+
+  nameOfTheAttribute:
+    tags: ["Bibliographic lookup"]
+    title: "name of the attribute"
+    description: "Can contain inline and block AsciiDoc formatting".
+
+  #--- Caches ---#
+      #--- Is there some reference information hidden in https://www.metanorma.org/author/topics/building/reference-lookup/#lookup-result-caching ? ---#
+
+  no-isobib-cache:
+    tags: ["Caches"]
+    title: "No isobib cache"
+    description: "If set, use the relaton and iev gem functionality to look up ISO and IEV references online, but do not use the cache of relaton and iev searches".
+
+  local-cache:
+    tags: ["Caches"]
+    title: "Local cache"
+    description: "Use the local relaton and iev search caches to override the global relaton and iev search caches. If a directory name is given for the attribute, that name overrides relaton as the cache name".
+
+  local-cache-only:
+    tags: ["Caches"]
+    title: "Local cache only"
+    description: |
+      "Use the local relaton and iev search caches to the exclusion of the global
+      `relaton` and `iev` search caches.
+      If a directory name is given for the attribute, that name overrides `relaton` as the cache name".
+
+  flush-caches:
+    tags: ["Caches"]
+    title: "Flush caches"
+    description: "If set, delete and reinitialise the cache of `https://www.relaton.com/[relaton]` searches".
+
+  #--- Math ---#
+      #--- Please include all references from https://www.metanorma.org/author/topics/document-format/text/#mathematical-expressions ---#
+  stem:
+    tags: ["Math"]
+    title: "Mathematical Markup Language"
+    description: "This attribute is required if you have any AsciiMath, MathML, or LaTeX included in the document; otherwise they will not be detected. If you want the STEM expressions in your document to be interpreted as LaTeX by default, use `:stem: latexmath`".
+    more_information: "Read more about link:/author/topics/document-format/text/#mathematical-expressions[mathematical expressions]."
+
+
+  #--- Languages and localization ---#
+      #--- Please include all references from https://www.metanorma.org/author/topics/languages/ ---#
+  i18nyaml:
+    tags: ["Languages and localization"]
+    title: "YAML language template"
+    description: |
+      "Name of YAML language template file.
+      Use if you wish to output an standard in a language that’s not supported out of the box.
+      For more on how to customise localization, see link:/builder/topics/localization[Localization].".
+
+  language:
+    tags: ["Languages and localization"]
+    title: "Language"
+    description: "Two-letter code (ISO 639-1) of the language the document is written in. Defaults to `en`".
+    value_default: en
+
+  script:
+    tags: ["Languages and localization"]
+    title: "Script"
+    description: "Script of the document (ISO 15924). Defaults to `Latn`. Must be supplied as `Hans` for Simplified Chinese".
```

Comment:
According to [wikipedia](https://en.wikipedia.org/wiki/ISO_15924), there is a large list of codes.
I'm not sure if we must put it into a `value_spec` key.
Maybe we can refer it to an external [link](https://en.wikipedia.org/wiki/ISO_15924) instead, @opoudjis?

Reply:
We have as of this writing only accounted for the following in code (when guessing scripts from language), and while it would be wonderful for a thousand flowers to bloom, I would be surprised if we end covering even them:

Arab: Arabic script
Aran: Urdu Arabic script
Cyrl: Cyrillic script
Deva: Devanagari script
Grek: Greek script
Hans: Chinese Simplified script
Kore: Korean script
Hebr: Hebrew script
Jpan: Japanese script
Latn: Latin script